libs/mocks/src/outbound_queue.rs

```diff
@@ -17,12 +18,20 @@ pub mod pallet {
 	#[pallet::storage]
 	type CallIds<T: Config> = StorageMap<_, _, String, mock_builder::CallId>;
 
+	#[pallet::storage]
+	type DomainAddressHook<T: Config> =
+		StorageMap<_, _, <T as Config>::Destination, <T as Config>::Sender>;
```

Comment:
Ahh it's because of that. I think this can be removed from the mock

Reply:
Copy-paste probably (?)

Reply:
Great catch! Yes, copy-paste and a blind eye.